pkgs/development/python-modules/importlib-metadata/default.nix

```diff
@@ -8,22 +8,23 @@
 , configparser
 , isPy3k
 , importlib-resources
+, toml
 , packaging
 }:
 
 buildPythonPackage rec {
   pname = "importlib-metadata";
-  version = "1.7.0";
+  version = "3.4.0";
```

Comment:
since 3.0.0. the project requires python 3.6 or later. I think we should mark this for a library with that many dependencies.

Reply:
Added, thanks for checking.

Reply:
did you remove the disabled again?

Reply:
More like not saved before committing.